### DISCUSSION TITLE
docker安装的如何设置用户名密码

### DISCUSSION THREAD

**Discussion by feifeify** [messense/aliyundrive-webdav#64] [Q&A] [answered]
docker安装的如何设置用户名密码，求教

**Comment by messense** [comment, marked as answer]
环境变量配置 WEBDAV_AUTH_USER 和 WEBDAV_AUTH_PASSWORD 比如

`-e WEBDAV_AUTH_USER=admin -e WEBDAV_AUTH_PASSWORD=admin`
